functional_tests/test_db_online_offline.py

```diff
@@ -18,7 +19,7 @@
 import logging
 log = logging.getLogger(lib.settings.LOGGER)
 
-NUM_ENDPOINTS = 10
+NUM_ENDPOINTS = 13
 
 def rest_scan(sync_gateway, db, online, num_docs, user_name, channels):
```

Comment:
The behavior of this should be idempotent. Test case should be able to use this function multiple times with true/false return value. or the signature should be changed to allow expected_num_docs/revisions etc.